### DISCUSSION TITLE
Importing markers from text file - XML creation from text file

### DISCUSSION THREAD

**Discussion by Artybartfast** [CommandPost/FCPCafe#402] [Final Cut Pro Questions]
I was given a list of edit points, simple time code and a description of each edit and I wondered if there was a simple program or way to get the file converted to XML and then import it into my project. Editing so little lately. that I'm probably not wanting to spend any serious money on this right now :( 

I asked the AI bots and they gave me an XML file but I see that FCPxml file is actually a package? Also didn't work. I ended up doing it manually as only 12 edit but would be nice to speed things up in the future. They also gave me some python code so that might be an option to mess about with.

**Comment by benblaser60** [comment]
FCPXML is not standard XML. There's no really easy way to convert it. One of FCP's shortcomings.
